kong/plugins/datadog/schema.lua

```diff
@@ -79,9 +80,13 @@ return {
           { service_name_tag = { type = "string", default = "name" }, },
           { status_tag = { type = "string", default = "status" }, },
           { consumer_tag = { type = "string", default = "consumer" }, },
-          { retry_count = { type = "integer", required = true, default = 10 }, },
-          { queue_size = { type = "integer", required = true, default = 1 }, },
-          { flush_timeout = { type = "number", required = true, default = 2 }, },
+          { retry_count = { type = "integer", required = true, default = 10 }, },  -- deprecated, use queue.max_retry_time
```

Comment:
These "deprecation comments" are not really enough. We need to:
* Make them an "alias" to the new queue fields (using shorthand_fields)
* Emit a deprecation warning when they are being used